src/mb_enhanced_cover_art_uploads/providers/amazon.ts

```diff
@@ -26,6 +27,15 @@ const VARIANT_TYPE_MAPPING: Record<string, ArtworkTypeIDs | undefined> = {
     // See https://sellercentral.amazon.com/gp/help/external/JV4FNMT7563SF5F for further details
 };
 
+// CSS queries to figure out which type of page we're on
+const AUDIBLE_PAGE_QUERY = '.audible_mm_title';
```

Comment:
Maybe we should keep my previous comment about the origin of this selector for this line? It would be good to know that this referred to the audiobook button with the Audible logo if Amazon decides to change this (not very meaningful) class name in the future.